src/i18n/gr/index.js

```diff
@@ -0,0 +1,157 @@
+import potato from './potato';
+import print from './print';
+import tester from './tester';
+export default {
+  el: {
+    message: {
+      ...potato,
+      help: {
+        label: 'Καλέστε τον Μάγο'
+      },
+      print: { ...print },
+      tester: { ...tester },
+      keyboard: {
+        label: 'πληκτρολόγιο'
+      },
+      layout: {
+        label: 'διάταξη'
+      },
+      keymapName: {
+        label: 'Όνομα Διάταξης',
+        placeholder: 'προσαρμοσμένο όνομα keymap'
+      },
+      loadDefault: {
+        label: 'Φότωση Προεπιλεγμένων',
```

Comment:
Φόρτωση ;

Reply:
Yes it should be φόρτωση, I must have missed it. Good catch yiancar!